### PR TITLE
Add Music

### PR DESCRIPTION
**Pull request by ciconia**
Didn't really know where to add this, so I created a new category called Creating, right after Gaming.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Put it in the Misc section. I only create new categories when there's at least two items that could go into it.

**Comment by sindresorhus** [comment]
Please read the [contributing guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [awesome manifest](https://github.com/sindresorhus/awesome/blob/master/awesome.md). The list is missing a license.

**Comment by ciconia** [comment]
Also, I added a license to https://github.com/ciconia/awesome-music.

File: readme.md [hunk 1]
```diff
@@ -221,7 +221,6 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 - [Unity](https://github.com/RyanNielson/awesome-unity) *(Game engine)*
 - [Chess](https://github.com/hkirat/awesome-chess)
 
-
```
Comment (sindresorhus) [inline readme.md]: Don't do unrelated changes.

**Comment by sindresorhus** [comment]
> The list should have a header `# Awesome Music`.

**Comment by ciconia** [comment]
Sorry for the messy commits, my poor git skills... :-)

**Comment by sindresorhus** [comment]
Added
